build.sbt

```diff
@@ -25,19 +25,35 @@ lazy val root = project
   .settings(name := "compendium")
   .settings(commonSettings)
   .settings(noPublishSettings)
-  .aggregate(server)
-  .dependsOn(server)
+  .aggregate(server, client)
+  .dependsOn(server, client)
+
+lazy val common = project
```

Comment:
what's this module going to do?

Reply:
It has common models, such as the protocol. In this way there are no dependencies between client and server.

Reply:
Cool!